src/metar/metar_csv2ioda.py

```diff
@@ -0,0 +1,322 @@
+        wvec = [None] * len(pvec)
+
+        n = 0
+        while n < len(pvec):
+            w = self.meteo_utils.r_sub_s(pvec[n]*1.0, tdvec[n]+273.15)
+            print ("DEBUG, w is: " + str(w))
+            wvec[n] = w
+            n = n + 1
+
+        pwat = self.meteo_utils.precipitable_water(pvec, wvec)
+        print ("DEBUG, precip water is: " + str(pwat))
+
+        z_test = self.meteo_utils.std_atmos(70000.0)
+        print ("DEBUG, z_test is: " + str(z_test))
+        p_test = self.meteo_utils.std_atmos_p(3000.0)
+        print ("DEBUG, p_test is: " + str(p_test))
+        x_test = self.meteo_utils.r_sub_s(90000.0, 275.15)
+        print ("DEBUG, x_test is: " + str(x_test))
+        y_test = self.meteo_utils.r_sub_i(90000.0, 270.15)
+        print ("DEBUG, y_test is: " + str(y_test))
+        '''
+
+        self.data = data
+
+        return
+
+class IODA(object):
+
+    def __init__(self, filename, date, varDict, obsList):
+        '''
+        Initialize IODA writer class,
+        transform to IODA data structure and,
+        write out to IODA file.
+        '''
+
+        self.filename = filename
+        self.date = date
+        self.varDict = varDict
+
+        self.locKeyList = [
+            ("station_id", "string"),
+            ("latitude", "float"),
+            ("longitude", "float"),
+            ("station_elevation", "float"),
+            ("datetime", "string")
+        ]
+
+        self.AttrData = {
+            'odb_version': 1,
+            'date_time_string': self.date.strftime("%Y-%m-%dT%H:%M:%SZ")
+        }
+
+        self.writer = iconv.NcWriter(self.filename, self.locKeyList)
+
+        self.keyDict = DefaultOrderedDict(lambda: DefaultOrderedDict(dict))
+        for key in self.varDict.keys():
+            value = self.varDict[key]
+            self.keyDict[key]['valKey'] = value, self.writer.OvalName()
+            self.keyDict[key]['errKey'] = value, self.writer.OerrName()
+            self.keyDict[key]['qcKey'] = value, self.writer.OqcName()
+
+        # data is the dictionary containing IODA friendly data structure
+        self.data = DefaultOrderedDict(lambda: DefaultOrderedDict(dict))
+
+        recKey = 0
+
+        for obs in obsList:
+
+            for n in range(len(obs.data['ob_lat'])):
+
+                icao = obs.data['ob_icao'][n]
+                lat = obs.data['ob_lat'][n]
+                lon = obs.data['ob_lon'][n]
+                elev = obs.data['ob_elev'][n]
+                dtg = obs.data['ob_datetime'][n]
+                locKey = icao, lat, lon, elev, dtg
+
+                print ("obs iterate: " + str(n) + ", " + icao + ", " + str(lat) + ", " + str(lon) + ", " + str(elev) + ", " + dtg)
```

Comment:
Is this much verbosity necessary for non-debugging?

Reply:
Agree completely. It was purely for debugging.  Similar to UFO (and all JEDI codes), is there just a ioda-converters standardized method of putting in something like oops_debug in order to get the debug prints only when a user sets an environment variable?

Reply:
I'm not sure about that, I don't know of one, but perhaps @srherbener does. I think it's unlikely because compiled vs interpreted code, so it's much easier to add/remove print statements in python scripts if you need to for quick debugging.

Reply:
I think I'll keep the print line present but comment it out.  For the purposes of NOAA-GSL, I need to add a bunch more variables for their DA (elements such as cloud base height, visibility, etc.) so I would like to keep it around as I make additions and later testing/debugging.  If another reviewer comments about a standardized way to print only when in a debug-mode, then I can adjust.